conan/tools/system/pip_manager.py

```diff
@@ -0,0 +1,45 @@
+import venv
+import platform
+import os
+
+from conan.errors import ConanException
+from conan.tools.env.environment import Environment
+
+
+class PipEnv:
+    accepted_install_codes = [0]
+    accepted_check_codes = [0, 1]
```

Comment:
```suggestion
```
this is not used any more, can we simplify the `accepted_install_codes` as well?